scala-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/scala/AbstractControlClient.scala

```diff
@@ -34,7 +34,7 @@ abstract class AbstractControlClient(conductrAddress: URL) {
    *         - BundleGetSuccess if the get bundle request has been succeeded. This object contains the bundle id, bundle file, and optionally the config file.
    *         - BundleGetFailure if the get bundle request has been failed. This object contains the HTTP status code and error message.
    */
-  def getBundle(bundleId: BundleId)(implicit cc: CC): Future[BundleGetResult]
+  def getBundle(bundleId: BundleId, bundleData: Subscriber[Array[Byte]], configData: Subscriber[Array[Byte]])(implicit cc: CC): Future[BundleGetResult]
```

Comment:
Params added - thanks for pointing this out.

I've added them into `ControlClient`, but I forgot to do so on the `AbstractControlClient`.